lib/features/sks_people_counter_button/models/sks_user_data.dart

```diff
@@ -0,0 +1,49 @@
+import "package:freezed_annotation/freezed_annotation.dart";
```

Comment:
TBH, you can disagree with me @tomasz-trela and @mikolaj-jalocha but imo I would rename the feature folder to just `sks_live_people` and place here both the button and the fullscreen chart too - because the model can be reused in the both subfeatures

Reply:
just a quick though, you can disagree - but how about smth like this? what you think guys?
```bash
features/
├─ sks_people_live/
│  ├─ data/
│  │  ├─ models/
│  │  ├─ repositories/
│  ├─ presentation/
│  │  ├─ live_button/
│  │  ├─ chart/
```

Reply:
It's good idea